### PR TITLE
add test cases for ssc multiple session test and cross boundary test

### PR DESCRIPTION
**Pull request by huanwwwq**

### CODE REVIEW

File: pike/test/copychunk.py [hunk 1]
```diff
@@ -826,3 +889,228 @@ def generic_mchan_ssc_test_case(self):
...
+            exp_error = pike.ntstatus.STATUS_SUCCESS
+        try:
+            with self.assert_error(exp_error):
+                result = future[0].result()
+        finally:
+            self.chan.close(fh_src)
+            self.chan.close(fh_dst)
+
+        self.assertIsInstance(
+            future[0].response.response.children[0].ioctl_output, pike.smb2.CopyChunkCopyResponse)
```
Comment (isi-mfurer) [inline pike/test/copychunk.py]: maybe to simplify this section of code you could get a reference to the ioctl response

```
ioctl_resp = future[0].response.response.children[0]
self.assertIsInstance(ioctl_resp.ioctl_output, pike.smb2.CopyChunkCopyResponse)
self.assertEqual(ioctl_resp.chunks_written, ssc_res[0])
self.assertEqual(ioctl_resp.chunk_bytes_written, ssc_res[1])
self.assertEqual(ioctl_resp.total_bytes_written, ssc_res[2])
```
